tests/lib/rules/no-invalid-regexp.js

```diff
@@ -288,6 +296,48 @@ ruleTester.run("no-invalid-regexp", rule, {
                 data: { message: "Invalid flags supplied to RegExp constructor 'z'" },
                 type: "NewExpression"
             }]
+        },
+
+        // ES2024
+        {
+            code: "new RegExp('[[]', 'v');",
+            errors: [{
+                messageId: "regexMessage",
+                data: { message: "Invalid regular expression: /[[]/u: Unterminated character class" },
+                type: "NewExpression"
+            }]
+        },
+        {
+            code: "new RegExp('.', 'uv');",
+            errors: [{
+                messageId: "regexMessage",
+                data: { message: "Regex 'u' and 'v' flags cannot be used together" },
+                type: "NewExpression"
+            }]
+        },
+        {
+            code: "new RegExp(pattern, 'uv');",
+            errors: [{
+                messageId: "regexMessage",
+                data: { message: "Regex 'u' and 'v' flags cannot be used together" },
+                type: "NewExpression"
+            }]
+        },
+        {
+            code: "new RegExp('[A--B]' /* valid only with `v` flag */, 'u')",
+            errors: [{
+                messageId: "regexMessage",
+                data: { message: "Invalid regular expression: /[A--B]/u: Range out of order in character class" },
+                type: "NewExpression"
+            }]
+        },
+        {
+            code: "new RegExp('[[]\\\\u{0}*' /* valid only with `u` flag */, 'v')",
+            errors: [{
+                messageId: "regexMessage",
+                data: { message: "Invalid regular expression: /[[]\\u{0}*/u: Unterminated character class" },
```

Comment:
I noticed that the error message issued by regexpp was wrong. i will fix the error message.

Reply:
Okay, waiting for the fix then. Otherwise, this LGTM.

Reply:
I fixed the error message.
https://github.com/eslint/eslint/pull/17404/commits/1a775c4f55d2892a335b6255dcf0e37c6f5852de